src/type/definition.js

```diff
@@ -1024,3 +1024,10 @@ export class GraphQLNonNull<T: GraphQLNullableType> {
     return this.ofType.toString() + '!';
   }
 }
+
+export type {
```

Comment:
Please write export at the definition of the type rather than exporting at the bottom of the file